### DISCUSSION TITLE
No such file or directory as: .../show_lldp_neighbors_detail.tpl

### DISCUSSION THREAD

**Discussion by remchuk** [netbox-community/netbox#13979] [Help Wanted!]
Hey there! So we have a Netbox in our on prem environment, and we recently set a Prometheus and we monitor differences in our environment and the Netbox.

For some reason many switches are show as unreachable.

When checking the logs I received:
```
Errorcode="textfsmExtractorErr01" in naplam.base.helpers with systemMessage="[Errno2] No such file or directory: '/use/local/lib/python3.11/site-packages/naplam/nxos_ssh/utils/textfsm_templates/show_lldp_neighbors_detail.tpl'"
```
Has someone had this issue?

**Comment by jcollie** [comment]
I don't know what's going on specifically, but "napalm" is misspelled as "naplam". Plus Napalm was removed from the core, did you add it back in as a plug-in, or are you running an older version?

**Reply by candlerb** [reply]
More importantly, the code is not running inside a virtualenv. It's under `/use` [sic] `/local/lib/python3.11/site-packages/...` which is where global packages are installed.

Copy-paste errors aside, I suspect this means it's an ancient installation of Netbox. upgrade.sh was changed to use virtualenv in [v2.7.9](https://github.com/netbox-community/netbox/releases/v2.7.9), released 6 Mar 2020.
